Less verbose warnings for unrecognized lines

dhalperi opened the pull request:
I noticed we print the line text twice -- that seems unnecessarily verbose.

Also explains why the warnings can't really be linked to a file.

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/batfish/batfish/884)
<!-- Reviewable:end -->

corinaminer commented:
Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

*[projects/batfish/src/main/java/org/batfish/grammar/cisco/CiscoControlPlaneExtractor.java, line 7603 at r1](https://reviewable.io:443/reviews/batfish/batfish/884#-L43cAbrkvUQeWuIfAcI:-L43cAbrkvUQeWuIfAcJ:b4ws9or) ([raw file](https://github.com/batfish/batfish/blob/a8ba4e52f85c58d68d830a3dde6318fc195977f6/projects/batfish/src/main/java/org/batfish/grammar/cisco/CiscoControlPlaneExtractor.java#L7603)):*
> ```Java
>     String msg = String.format("Unrecognized Line: %d: %s", line, lineText);
>     if (_unrecognizedAsRedFlag) {
>       _w.redFlag(msg + " SUBSEQUENT LINES MAY NOT BE PROCESSED CORRECTLY\n");
> ```

old message didn't terminate with `\n` -- will adding a new line char now cause these warnings to print with an extra blank line between them?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/batfish/batfish/884)*
<!-- Sent from Reviewable.io -->

dhalperi commented:
*[projects/batfish/src/main/java/org/batfish/grammar/cisco/CiscoControlPlaneExtractor.java, line 7603 at r1](https://reviewable.io:443/reviews/batfish/batfish/884#-L43cAbrkvUQeWuIfAcI:-L450oju7NO2reWtbXQh:b-ho6au4) ([raw file](https://github.com/batfish/batfish/blob/a8ba4e52f85c58d68d830a3dde6318fc195977f6/projects/batfish/src/main/java/org/batfish/grammar/cisco/CiscoControlPlaneExtractor.java#L7603)):*
<details><summary><i>Previously, corinaminer (Corina Miner) wrote…</i></summary><blockquote>

old message didn't terminate with `\n` -- will adding a new line char now cause these warnings to print with an extra blank line between them?
</blockquote></details>

Good catch. Fixed, and updated ref shows the result.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/batfish/batfish/884)*
<!-- Sent from Reviewable.io -->

corinaminer commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

---

Reviewed 2 of 2 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/batfish/batfish/884#-:-L47XcoBa737UzfTy_N9:bnfp4nl)*
<!-- Sent from Reviewable.io -->
